### ISSUE TITLE
Code

### ISSUE THREAD

**Issue by zouyingcao** [InvokerStark/OverKill#1]
hello, authors.
I am interested in the implementation details, particularly regarding the rewritten "model.generate" function (I guess). 
If it is appropriate, may I ask when you plan to release the code?
Best regards,

**Comment by InvokerStark** [comment]
I'm glad you took an interest in our work. The code will be uploaded after the final version is submitted (maybe). In fact, we did not rewrite the **model.generate** function. You can simply try to call **model.forward** function to get the logits and then proceed directly.

**Comment by zouyingcao** [comment]
> I'm glad you took an interest in our work. The code will be uploaded after the final version is submitted (maybe). In fact, we did not rewrite the **model.generate** function. You can simply try to call **model.forward** function to get the logits and then proceed directly.

thanks for reply, wish opensource soon.

**Comment by jc-ryan** [comment]
@InvokerStark Excuse me, since another few months have passed since the submission of the final version of the paper, when will the code be updated?
